### PR TITLE
Removing the catalog and ingestion url in the recipe code

### PR DESCRIPTION
**Pull request by sathyarun18**
- The service has been updated to pass in the python catalog and ingestion url 
- Tested on prod and dev environments

### CODE REVIEW

File: recipes/python/retail/Dockerfile [hunk 1]
```diff
@@ -30,6 +30,6 @@ COPY . .
 
 RUN python setup.py clean --all install
 
-RUN cp dist/retail*.egg /application.egg
+COPY dist/retail*.egg /application.egg
```
Comment (jonaphin) [inline recipes/python/retail/Dockerfile]: I think we agreed to use "RUN cp" as we do not copy the egg file from local machine any longer. @shivmpatel12, can you please confirm or deny?

**Review by jonaphin — CHANGES_REQUESTED** [review]

File: recipes/python/retail/Dockerfile (hunk 1 above)
Comment (shivmpatel12) [reply recipes/python/retail/Dockerfile]: Yup confirmed. `RUN cp dist/retail*.egg /application.egg` because the egg is generated on the container

Comment (sathyarun18) [reply recipes/python/retail/Dockerfile]: It did not work with RUN cp whenever I try to generate the docker image locally.

**Review by tumulurik — APPROVED** [review]
LGTM!

File: recipes/python/retail/Dockerfile (hunk 1 above)
Comment (sathyarun18) [reply recipes/python/retail/Dockerfile]: This seems to happen when the egg file is generated locally. Reverting the changes.

**Review by jonaphin — APPROVED** [review]

File: recipes/python/retail/retail/datasaver.py [hunk 2]
```diff
@@ -21,12 +21,8 @@ def save(configProperties, prediction):
...
-    catalog_url = "https://platform.adobe.io/data/foundation/catalog"
-    ingestion_url = "https://platform.adobe.io/data/foundation/import"
-
-    writer = DataSetWriter(catalog_url=catalog_url,
-                           ingestion_url=ingestion_url,
-                           client_id=configProperties['ML_FRAMEWORK_IMS_USER_CLIENT_ID'],
+   
+    writer = DataSetWriter(client_id=configProperties['ML_FRAMEWORK_IMS_USER_CLIENT_ID'],
                            user_token=configProperties['ML_FRAMEWORK_IMS_TOKEN'],
                            service_token=configProperties['ML_FRAMEWORK_IMS_ML_TOKEN'])
```
Comment (rjrussell77) [inline recipes/python/retail/retail/datasaver.py]: If you have time, can you run `pylint` if you haven't already?  We are standardizing on this tool for all python code.  Here is my output, (not all things need fixing)
```
No config file found, using default configuration
************* Module datasaver
C: 24, 0: Trailing whitespace (trailing-whitespace)
C: 38, 0: Exactly one space required after comma
        },  index=prediction.index))
         ^ (bad-whitespace)
C:  1, 0: Missing module docstring (missing-docstring)
E: 17, 0: Unable to import 'data_access_sdk_python.writer' (import-error)
C: 20, 0: Invalid argument name "configProperties" (invalid-name)
C: 20, 0: Missing function docstring (missing-docstring)
C: 31, 4: Invalid variable name "tenantId" (invalid-name)
```

**Review by rjrussell77 — APPROVED** [review]
LGTM after reviewing/fixing any pylint-raised concerns

**Pull request merged**